Tests/XCUITests/IntegrationTests.swift

```diff
@@ -231,7 +238,6 @@ class IntegrationTests: BaseTestCase {
         app.buttons["urlBar-cancel"].tap()
         // Sign into Firefox Accounts
         signInFxAccounts()
-        sleep(3)
```

Comment:
We don't need a `sleep()` before calling `waitForInitialSyncComplete()` in other tests. :-/